test/libsolidity/syntaxTests/string/string_unicode_without_prefix.sol

```diff
@@ -4,4 +4,4 @@ contract test {
     }
```

Comment:
Just as @chriseth said, this message could also be printed for example for [control characters](https://en.wikipedia.org/wiki/Control_character). Could you add a test that includes some?